sunpy/database/database.py

```diff
@@ -378,6 +378,17 @@ def _download_and_collect_entries(self, query_result, client=None,
         if client is None:
             client = VSOClient()
 
+        remove_list = []
+        for qr in query_result:
+            temp =  tables.DatabaseEntry._from_query_result_block(qr)
+            for database_entry in self.__iter__():
+                if database_entry.path is not None and temp.__compare_attributes__(database_entry, ["source", "provider", "physobs", "fileid", "observation_time_start", "observation_time_end", "instrument", "size", "wavemin", "wavemax"]):
```

Comment:
can you wrap this onto multiple lines, try to stick to 79 or 99 character lines.